indico/modules/events/contributions/models/contributions.py

```diff
@@ -528,6 +529,10 @@ def has_published_editables(self):
     def slug(self):
         return slugify(self.friendly_id, self.title, maxlen=30)
 
+    @property
+    def show_title(self):
+        return persons_settings.get(self.event, 'show_title')
```

Comment:
This should not be a property on the Contribution model since it's an event-wide setting. Having it as a property on the event would make much more sense...

Reply:
thanks that was my confusion too

Reply:
are you ok with show_title instead of hide_title?

Reply:
tbh i preferred the old one, but i don't haev a super strong opinion

Reply:
Done